src/routes/major.py

```
@@ -8,62 +8,68 @@
 
-@major_router.post('/executive/major/create', status_code=201)
+@major_router.post("/executive/major/create", status_code=201)
 async def create_major(session: SessionDep, body: BodyCreateMajor) -> Major:
     major = Major(college=body.college, major_name=body.major_name)
     session.add(major)
-    try: session.commit()
+    try:
+        session.commit()
     except IntegrityError:
         session.rollback()
         raise HTTPException(409, detail="major already exists")
```

Ruff (0.13.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)